src/components/HomePage/SearchArea/SearchArea.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'react-redux';
+import { addSnackbar } from '../../../actions/notificationActions';
+
+import JOB_TYPES from './jobTypes';
+
+import FormControl from '@material-ui/core/FormControl';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import Slider from '@material-ui/core/Slider';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import Collapse from '@material-ui/core/Collapse';
+
+import { makeStyles } from '@material-ui/styles';
+import searchAreaStyle from "./SearchArea.module.css";
+
+import SearchBar from "./SearchBar";
+import ShowAdvancedOptionsButton from './ShowAdvancedOptionsButton';
+import SliderValueTooltip from './SliderValueTooltip';
+
+const INITIAL_JOB_TYPE = "";
+const INITIAL_JOB_DURATION = 1;
+
+const useStyles = makeStyles({
+    wrapperInner: {
+        display: "grid",
+        "grid-template-columns": "1fr 1fr",
+        "grid-template-rows": "1fr",
+        "align-items": "center",
+        "grid-gap": "1em",
+    }
+});
+
+function SearchArea(props) {
+
+    const {addSnackbar, onSubmit} = props;
+    const classes = useStyles();
+    
+    // Set initial form values
+    const [searchValue, setSearchValue] = React.useState("");
+    const [advancedSearch, setAdvancedSearch] = React.useState(false);
+    const [jobType, setJobType] = React.useState(INITIAL_JOB_TYPE);
+    const [jobDuration, setJobDuration] = React.useState(INITIAL_JOB_DURATION);
```

Comment:
Import the hook directly? Personal preference, tbh, but would rather always use the hooks without the trailing `React` - will also look better when mixing with custom hooks or hooks from other sources (I spy one just above, `useStyles`, that looks out of place due to this)

Reply:
I think we should use `React.` for the vanilla hooks, and for the custom ones, such as `useStyles()`, omit it of course

Reply:
Simply put, really don't see any point for wasting characters writing `React.` everywhere :/
Can see the point you make for it, but think that the name and import already distinguishes the hook as a vanilla one (if someone tries to create custom hooks with the same name as any of the vanilla ones, I will personally nuke their PRs, _no problemo_).

This would still make hooks recognisable as `^use.*` (`useStuff`, `useSomething`), and thus the linting from [eslint-plugin-react-hooks](https://www.npmjs.com/package/eslint-plugin-react-hooks) would still apply so don't really see any point in keeping this 😕